### PR TITLE
US93694: add icons for grades page

### PR DESCRIPTION
**Pull request by tinglisD2L**
https://rally1.rallydev.com/#/15545269080/detail/userstory/191201794100

### CODE REVIEW

**Comment by dbatiste** [comment]
Looking at the icons... I'm really not sure I can tell what they are for (names aside).

**Comment by tinglisD2L** [comment]
@njostonehouse I renamed the icon.
@dbatiste I can't comment on the designs because I did not create them. @nickdingle is the designer behind them.

**Comment by dbatiste** [comment]
I imagine context helps.  The visible and remove part is obvious to me.  The other part of the icon doesn't mean anything to me.  The exempt one depends completely on context for meaning.

**Comment by tinglisD2L** [comment]
I think the exempt icon is meant to bypass something, whereas the unexempt one doesn't bypass anything.

File: images/tier1/unexempt.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg width="18" height="19"  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```
Comment (dlockhart) [inline images/tier1/unexempt.svg]: Should be 18x18, not 18x19.

**Comment by dlockhart** [comment]
Could you put these through [SVGOMG](https://jakearchibald.github.io/svgomg/) to clean them up a bit? I'm seeing ~30% reduction in size after putting one of them through it. You'll need to put the `viewBox` back on afterwards as it strips it.

File: images/tier1/grade-visible.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21 18">
```
Comment (dlockhart) [inline images/tier1/grade-visible.svg]: The viewbox needs to be `0 0 18 18`. To get that to work though you'll need to add a transform that moves it `3px` and then SVGOMG can optimize it. I can give you a hand with fixing this.

**Comment by dlockhart** [comment]
I'd like to see `exempt` and `unexempt` grouped somehow alphabetically so it's obvious they're related. Is there something that relates them together that the user is exempted from? e.g. `voting-exempt`, `autograde-exempt`, etc.

**Comment by tinglisD2L** [comment]
@dlockhart In this case, it's `grade-exempt` and `grade-unexempt` but it may be reused for Content as well in the future. I suppose it's possible to name them `exempt-add` and `exempt-remove` for the sake of grouping. Let me know what you decide on.

File: images/tier1/grade-visible.svg (hunk 2 above)
Comment (dlockhart) [reply images/tier1/grade-visible.svg]: I've reached out to Nick to try and get this resolved, so ignore for now. :)

**Comment by tinglisD2L** [comment]
@dlockhart 
For the icon with `viewBox="0 0 21 18"`, translating the paths then adjusting the viewBox seems to result in truncating. So instead I scaled the icon by 18/21 and then added translate(0,1) to make it vertically centre again. Here's what I have. Is it what you want it to look like? (I put in the bounding box to show the position better. I'll remove it after)

**Comment by dlockhart** [comment]
@tinglisD2L thanks for taking a shot at fixing the viewBox -- I ended up in the same place as you when I tried, so I reached out to Nick to see what he'd like to do. It's probably just a mistake with the SVG.

I like `exempt-add` and `exempt-remove` or maybe `exemption-add` and `exemption-remove` to make it sounds more like the icon is meant to add or remove an exemption?

**Comment by tinglisD2L** [comment]
Besides the funny business with `grade-visible.svg`, everything else has been addressed. I'm just waiting on Nick's reply.

**Pull request merged**